jina/docker/hubio.py

```diff
@@ -532,7 +534,7 @@ def _freeze_jina_version(self) -> None:
                     fp.write('\n'.join(new_requirements))
 
     def _check_completeness(self) -> Dict:
-        dockerfile_path = get_exist_path(self.args.path, 'Dockerfile')
+        self.dockerfile_path = get_exist_path(self.args.path, 'Dockerfile')
```

Comment:
please, do not set an attribute at this level.

Reply:
Why not? A few lines bellow a couple of other attributes are set

Reply:
Yes but something to be avoided

Reply:
With best effort attributes should be acquired in init time. Otherwise thr patterns of if hasattr() appear.

Reply:
Ok, I agree with you here. 

However, moving all attribute setting to init would require a more thorough redesign of this module (no fewer than 5 other attributes are set in `_check_completeness` function), which I think is out of scope of this PR - I am just addressing one issue within the current framework.

Reply:
Okey lets keep it here, I would just avoid not to add yet another one

Reply:
Understood. Technically I am just replacing setting `revised_dockerfile_path` attr (which appeared originally later in the same function) with `dockerfile_path`, so net 0 of attributes are set :)